### ISSUE TITLE
sitewide 508-defect-3 [COGNITION, SCREENREADER, SEMANTIC MARKUP]: The TTY number SHOULD not include TTY in the aria-label

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#16178]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

```diff
! Team affected: BAM1; Project: HLR
```

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

<hr/>

## Point of Contact

**VFS Point of Contact:** _Jennifer_

## User Story or Problem Statement

As a screen reader user, I expect to hear the label TTY followed by the linked number.

## Details

In the 711 implementation of the telephone link pattern, the `aria-label` currently includes "TTY" — although there is a text label of "TTY:" preceding it. This means a screen reader user will hear, "TTY: link TTY 7 1 1", which adds extra noise and cognitive friction. The aria-label serves as the link text for assistive text, so the WCAG success criteria listed under WCAG Guidance applies.

## Acceptance Criteria

- [ ] The TTY link text does not repeat the TTY of the text label.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: all
* Server destination: all environments

## Steps to Recreate

1. Enter `<https://staging.va.gov/decision-reviews/higher-level-review/request-higher-level-review-form-20-0996/introduction>` in browser
2. Start screenreading device listed in Environment
3. Navigate to the Need help? section at the bottom of the page
4. Verify that the screen reader repeats TTY.

### Current

``` html
<p class="help-talk">
  Need help filling out the form or have questions about eligibility? Please call VA Benefits and Services at <a class="no-wrap " href="tel:+18008271000" aria-label="8 0 0. 8 2 7. 1 0 0 0.">800-827-1000</a>.<br><br>If you have hearing loss, call TTY: <a class="no-wrap " href="tel:+1711" aria-label="TTY: 7 1 1.">711</a>.
</p>
```

### Recommendation

``` diff
<p class="help-talk">
  Need help filling out the form or have questions about eligibility? Please call VA Benefits and Services at <a class="no-wrap " href="tel:+18008271000" aria-label="8 0 0. 8 2 7. 1 0 0 0.">800-827-1000</a>.
</p>
+ <!-- separated into individual paragraphs for semantics -->
<p class="help-talk">
+ <!-- removed the "TTY:" from the aria-label -->
  If you have hearing loss, call TTY: <a class="no-wrap " href="tel:+1711" aria-label="7 1 1.">711</a>.
</p>
```

## WCAG or Vendor Guidance (optional)

* [WCAG 2.0 Level A - 2.4.4 – Link Purpose (In Context)](https://www.wuhcag.com/link-purpose-in-context/)

* [WCAG 2.0 Level AAA - 2.4.9 – Link Purpose (Link Only)](https://www.wuhcag.com/link-purpose-link-only/)

## Screenshots or Trace Logs

![Screen Shot 2020-11-13 at 9 34 52 AM](https://user-images.githubusercontent.com/57469/99123183-af081d00-25cd-11eb-956b-f99d5c33ce2f.png)

**Comment by 1Copenut** [comment]
@jenstrickland Captured this in the Design System repo upstream: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/218

**Comment by Mottie** [comment]
This was fixed in https://github.com/department-of-veterans-affairs/vets-website/pull/15043.

**Comment by joshkimux** [comment]
<img width="525" alt="Screen Shot 2021-01-05 at 11 51 18 AM" src="https://user-images.githubusercontent.com/14154792/103674087-552c2f80-4f4c-11eb-9294-482001d82c5d.png">
Closing!

**Comment by joshkimux** [comment]
Confirmed this has been fixed in HLR-- should we close out this ticket or leave it open as it's still sitewide?

**Comment by Mottie** [comment]
That's up to you... it's definitely a site-wide issue that needs to get fixed in the Telephone component.

**Comment by joshkimux** [comment]
@1Copenut with the thought of creating an automatic health dashboard, we may want to talk about how we handle sitewide tickets. Should we keep these open/closed given that a design system ticket has been written too?

**Comment by 1Copenut** [comment]
@joshkimux I agree, we should be mindful of making sure we're not doubling up issues. Here's my initial thought:

* If a design system issue is identified in an app, we should keep the app issue open and open a second issue in the design system repo. The app issue should have a comment and link to the design system issue, and the app epic should not be closed. This may cause some confusion or push back, but I think we're losing issues between groups sometimes.
* The design system issue URL should be copy/pasted into the app issue Description, so there's a link created between the two.
* If it's a sitewide that lives outside the design system, let's write a new issue on this repo, and add the label `sitewide` to it.

**Comment by laflannery** [comment]
Closing - I confirmed the specific issue in the ticket has been resolved and the DS component has been updated to remove the aria-label
